src/cascadia/TerminalSettingsModel/Command.cpp

```diff
@@ -46,10 +46,11 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         // TODO GH#6900: We probably want ActionAndArgs::Copy here
         //              This is fine for now because SUI can't actually
         //              modify the copy yet.

```

Comment:
this comment is now sus

Reply:
Actually, it's still valid! When we create SUI, we're still passing the original `ActionAndArgs` over 😳. But this is ok to do for now because SUI has no way of modifying actions yet. And modifying the settings.json completely reloads SUI and TermApp, so we can just push this off until `Actions.xaml` isn't a read-only page.

Reply:
Derp. Nevermind. Fixed.